packages/cli/src/editor/helpers/use-menu-structure.tsx

```diff
@@ -151,6 +151,34 @@ export const useMenuStructure = (closeMenu: () => void) => {
 						subMenu: null,
 						quickSwitcherLabel: 'New still...',
 					},
+					{
+						id: 'open-in-editor',
+						value: 'open-in-editor',
+						label: 'Open in code editor...',
+						onClick: async () => {
+							await fetch(`/api/open-in-editor`, {
```

Comment:
Now duplicated with the code in `OpenInEditor`, can we create a helper function to be used in both places?